lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -114,7 +114,7 @@ def create_new_region_questions(warn = true)
       raise MiqSignalError if warn && !agree(CREATE_REGION_AGREE)
     end
 
-    def ask_for_database_credentials
+    def ask_for_database_credentials(password_once = true)
```

Comment:
I think I would rather default to the previous behavior, then change only the callers that were causing the issue before.